spec/requests/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns http success and check correct placeholder text.' do
+      get '/'
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to include('Here is a list of Users')
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success and check correct placeholder text.' do
+      get '/users/:id'
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to include('Here is a specific user')
+    end
+  end
+end
```

Comment:
- [ ] kindly add a test that ensures the correct view template is rendered.
Example :` expect(response).to render_template(:show)`